src/Conticket/Model/Events/Event/EventWasCreated.php

```diff
@@ -0,0 +1,42 @@
+<?php
+/*
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
+ * "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT
+ * LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR
+ * A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT
+ * OWNER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL,
+ * SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT
+ * LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE,
+ * DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY
+ * THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+ * (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+ * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ *
+ * This software consists of voluntary contributions made by many individuals
+ * and is licensed under the MIT license.
+ */
+namespace Conticket\Model\Events\Event;
+
+use Conticket\Model\Aggregates\Event\EventId;
+use Prooph\EventSourcing\AggregateChanged;
+
+class EventWasCreated extends AggregateChanged
+{
+    public static function fromEventAndNameAndDescription(EventId $eventId, $name, $description)
+    {
+        return self::occur($eventId->toString(), [
+            'name' => $name,
+            'description' => $description
```

Comment:
You should, let me explain you why... When you don't put it and you had to append a thing to the array, you are affection two line, your `git diff` will looks like the follow:

```diff
- 'description' => $description
+ 'description' => $description,
+ 'other' => $thing,
```

When you have this final comma, you'll just affect the line that really needed to be changed:

```diff
+ 'other' => $thing,
```

💃